beacon_node/beacon_processor/src/metrics.rs

```diff
@@ -221,6 +245,15 @@ pub static BEACON_PROCESSOR_REPROCESSING_QUEUE_MATCHED_ATTESTATIONS: LazyLock<Re
             "Number of queued attestations where as matching block has been imported.",
         )
     });
+// TODO: This should be labeled instead of N single metrics
+pub static BEACON_PROCESSOR_REPROCESSING_QUEUE_MATCHED_SAMPLING_REQUESTS: LazyLock<
+    Result<IntCounter>,
+> = LazyLock::new(|| {
+    try_create_int_counter(
+        "beacon_processor_reprocessing_queue_matches_sampling_requests",
```

Comment:
The variable name doesn't match the metric name, not sure which was intended
```suggestion
        "beacon_processor_reprocessing_queue_matched_sampling_requests",
```